Description: Use system includes for lua
 Instead of hunting down the lua include files use the fixed place we
 install them from the package.
Author: Craig Small <csmall@debian.org>
Last-Update: 2017-03-17
--- a/src/TForkedProcess.h
+++ b/src/TForkedProcess.h
@@ -27,9 +27,9 @@
 
 extern "C"
 {
-    #include "lua.h"
-    #include "lualib.h"
-    #include "lauxlib.h"
+    #include <lua.h>
+    #include <lualib.h>
+    #include <lauxlib.h>
 }
 class TLuaInterpreter;
 
--- a/src/TLuaInterpreter.cpp
+++ b/src/TLuaInterpreter.cpp
@@ -46,9 +46,9 @@
 
 extern "C"
 {
-    #include "lua.h"
-    #include "lualib.h"
-    #include "lauxlib.h"
+    #include <lua.h>
+    #include <lualib.h>
+    #include <lauxlib.h>
 }
 #include <phonon>
 /*//for map var access
